ledger.go

```diff
@@ -86,17 +80,14 @@ func (e EvmosSECP256K1) GetAddressPubKeySECP256K1(hdPath []uint32, hrp string) (
 		return []byte{}, "", errors.New("unable to derive Ledger address, please open the Ethereum app and retry")
```

Comment:
Nit: we could return `nil` instead of an empty slice
```suggestion
		return nil, "", errors.New("unable to derive Ledger address, please open the Ethereum app and retry")
```

Reply:
cc @MalteHerrmann